docs/cloud/satellites/best-practices.md

```diff
@@ -0,0 +1,90 @@
+# Satellites Best Practices
+
+Earthly Satellites are remote runner instances provided by the Earthly team, offering the ability to perform builds in the cloud while retaining cache between runs. This best practices guide aims to help users optimize their workflow and leverage the full potential of Earthly Satellites.
+
+## Size Satellites for Builds
+
+Properly sizing Earthly Satellites is crucial to achieving optimal performance and resource utilization. Consider the following when determining which Satellite size is appropriate for your use case.
+
+### 1. Resource Requirements of a Single Build
+
+Understand the resource demands of your build, including memory usage, CPU requirements, and required cache size on disk. 
+
+For example, builds utilizing computationally intensive compilation steps, or large integration tests may require additional CPU and memory. Builds that download many gigabytes of dependencies may require a large amount of cache on disk.
+
+### 2. Concurrent Build Capacity
+
+Take into account the number of simultaneous builds that are likely to be run by your development team. The Satellite size should be chosen to handle the expected concurrency level effectively. 
+
+One metric to consider is the number of commits made per hour that will trigger builds on the satellite. For example, an xsmall or small satellites might handle a couple of concurrent builds, however, larger teams contributing more frequently may require a larger instance to accommodate the extra load.
+
+## Splitting Builds and Using More Satellites
+
+Large projects will often benefit from using multiple satellites in a single CI run, balancing the load across different machines in parallel. With this strategy, it’s best to optimize for the reuse of cache. Running the same target on the same satellite is more effective than alternating runs of the same target across different satellites.
```

Comment:
```suggestion
Large projects will often benefit from using multiple satellites in a single CI run, balancing the load across different machines in parallel. With this strategy, it’s best to optimize for cache reuse, by running the same target on the same satellite rather than alternating runs of the target across different satellites.
```